File: project/springboot/open-api/src/main/java/com/openapi/domain/dto/http/resonse/UserAuthResponse.java
package com.openapi.domain.dto.http.resonse;

import lombok.Data;

/**
 * 登录/注册后返回的用户认证信息
 */
@Data
public class UserAuthResponse {
    private Long userId;
    private String account;
    private String name;
    private String avatarUrl;
    private String accessToken;
}

File: demo/kmp/shared/src/commonMain/kotlin/com/vectordemo/domain/dto/http/response/UserAuthResponse.kt
package com.vectordemo.domain.dto.http.response

import kotlinx.serialization.Serializable

@Serializable
data class UserAuthResponse(
    val userId: String? = null,
    val account: String? = null,
    val name: String? = null,
    val avatarUrl: String? = null,
    val accessToken: String? = null,
)
